.examples/docker-compose/with-nginx-proxy/mariadb-cron-redis/fpm/web/nginx.conf

```diff
@@ -68,17 +68,22 @@ http {
             access_log off;
         }
 
-        # The following 2 rules are only needed for the user_webfinger app.
-        # Uncomment it if you're planning to use this app.
-        #rewrite ^/.well-known/host-meta /public.php?service=host-meta last;
-        #rewrite ^/.well-known/host-meta.json /public.php?service=host-meta-json last;
-
-        # The following rule is only needed for the Social app.
-        # Uncomment it if you're planning to use this app.
-        #rewrite ^/.well-known/webfinger /public.php?service=webfinger last;
-
-        location = /.well-known/carddav {
-            return 301 $scheme://$host/remote.php/dav;
+        # Make a regex exception for `/.well-known` so that clients can still
+        # access it despite the existence of the regex rule
+        # `location ~ /(\.|autotest|...)` which would otherwise handle requests
+        # for `/.well-known`.
+        location ^~ /.well-known {
+            # The following 6 rules are borrowed from `.htaccess`
+    
+            rewrite ^/\.well-known/host-meta\.json  /public.php?service=host-meta-json  last;
+            rewrite ^/\.well-known/host-meta        /public.php?service=host-meta       last;
+            rewrite ^/\.well-known/webfinger        /public.php?service=webfinger       last;
+            rewrite ^/\.well-known/nodeinfo         /public.php?service=nodeinfo        last;
+    
+            location = /.well-known/carddav     { return 301 /remote.php/dav/; }
+            location = /.well-known/caldav      { return 301 /remote.php/dav/; }
+    
+            try_files $uri $uri/ =404;
         }
 
         location = /.well-known/caldav {
```

Comment:
Section should be removed as well

Reply:
@0x47 well spotted. My mistake. Update done